docs/guides/styling.md

```diff
@@ -604,23 +604,22 @@ npm add -D concurrently
 
 ## CSS-in-JS libraries
 
-You can use CSS-in-JS libraries like Styled Components. Some of them require a "double render" in order to extract the styles from the component tree during the server render. It's unlikely this will effect performance in a significant way, React is pretty fast.
+You can use CSS-in-JS libraries like Styled Components. Some of them require a "double render" in order to extract the styles from the component tree during the server render. It's unlikely this will affect performance in a significant way, React is pretty fast.
 
 Here's some sample code to show how you might use Styled Components with Remix:
 
 1. First you'll need some context to put your styles on so that your root route can render them.
 
    ```tsx filename=app/StylesContext.tsx
-   // app/StylesContext.tsx
    import { createContext } from "react";
-   export default createContext<null | string>(null);
+   export default createContext<null | React.ReactElement>(
+     null
+   );
    ```
 
 2. Your `entry.server.tsx` will look something like this:
 
-   ```tsx filename=entry.server.tsx lines=6,7,16,20-26,29-30,35,37
```

Comment:
I couldn't see anywhere the `lines=` values were used in the docs, but I'm happy to bring them back if I'm missing something.

Reply:
oops, they're supposed to be inside of `[]` like `lines=[...]`, you mind adding that so it's easier to see where the relevant code changes go?

Reply:
Great, can do. I couldn't find docs on how to run the docs site locally — is that possible? I'd like to verify that I am counting lines correctly.

Reply:
nah, haven't opened up that repo yet, if you're off we'll fix 'em up later :)

Reply:
Gotcha. Will do.